src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
+/* eslint-disable function-paren-newline */
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const employeesObj = [];
+
+function prepareNumber(param) {
+  return param.replace('$', '').replace(',', '');
+}
+
+employees.forEach((e) =>
+  employeesObj.push({
+    name: e.textContent.trim(),
+    position: e.getAttribute('data-position'),
+    salary: prepareNumber(e.getAttribute('data-salary')),
+    age: e.getAttribute('data-age'),
```

Comment:
you can use [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) object instead of using each attribute separately